templates/project/.php-cs-fixer.dist.php

```diff
@@ -27,44 +27,31 @@
 HEADER;
 
 $rules = [
-    '@PHP70Migration' => true,
-    '@PHP70Migration:risky' => true,
-    '@PHP71Migration' => true,
-    '@PHP71Migration:risky' => true,
-    '@PHPUnit57Migration:risky' => true,
-    '@PHPUnit60Migration:risky' => true,
+    '@PHP74Migration' => true,
+    '@PHP74Migration:risky' => true,
+    '@PSR12' => true,
+    '@PSR12:risky' => true,
     '@Symfony' => true,
     '@Symfony:risky' => true,
-    'array_syntax' => ['syntax' => 'short'],
     'combine_consecutive_issets' => true,
     'combine_consecutive_unsets' => true,
-    'compact_nullable_typehint' => true,
     'global_namespace_import' => ['import_classes' => false, 'import_constants' => false, 'import_functions' => false],
     'header_comment' => ['header' => $header],
     'list_syntax' => ['syntax' => 'short'],
-    'logical_operators' => true,
     'method_argument_space' => ['on_multiline' => 'ensure_fully_multiline'],
     'multiline_whitespace_before_semicolons' => ['strategy' => 'no_multi_line'],
-    'no_extra_blank_lines' => true,
-    'no_php4_constructor' => true,
-    'no_superfluous_phpdoc_tags' => ['allow_mixed' => true],
     'no_useless_else' => true,
     'no_useless_return' => true,
     'nullable_type_declaration_for_default_null_value' => ['use_nullable_type_declaration' => true],
     'ordered_class_elements' => true,
     'ordered_imports' => ['sort_algorithm' => 'alpha', 'imports_order' => ['class', 'function', 'const']],
-    'php_unit_method_casing' => false,
-    'php_unit_set_up_tear_down_visibility' => true,
     'php_unit_strict' => true,
-    'php_unit_test_annotation' => false,
```

Comment:
Include in Symfony with true. It remove `@test`.
Looking at our test we should use it.